Source/mainmatter/IHL.tex

```diff
@@ -0,0 +1,102 @@
+\calculusName{Intuitionistic Hybrid Logic}   
+\calculusAcronym{\IHL}     
+\calculusLogic{Intuitionistic Logics} 
+\calculusLogicOrder{Propositional}
+\calculusType{Natural Deduction}   
+\calculusYear{2017}   
```

Comment:
@heades @vcvpaiva , I'm confused about the year when this calculus was invented. Here it says 2017, but the cited papers are from 2003, 2006 and 2011. Could you please clarify?